code/game/objects/items/weapons/implant/implants/carrion/carrion_spiders.dm

```diff
@@ -98,3 +103,12 @@
 
 /obj/item/implant/carrion_spider/proc/update_owner_mob()
 	owner_mob = owner_core.owner
+
+/obj/item/implant/carrion_spider/proc/toggle_group(group)
+	if(group == 1)
```

Comment:
Extra checks don't really affect anything, but consider doing `switch` or `else if`'s

Reply:
you already did that for me, so. Gonna resolve this and thanks